### ISSUE TITLE
Adoption of final IDDES simulation setup and obtaining buffet onset angle of attack for constant Ma and Re numbers for NACA 0012-34 airfoil

### ISSUE THREAD

**Issue by Tushargh29** [Tushargh29/transonic_shock_buffet#24]
Hello sir,

I did adopt the final IDDES simulation case setup of
**Reduced Y Grading of 5000, using wall functions and 450 cells chordwise** for **NACA 0012-34** airfoil.

Then, I tried to obtained the onset buffet angle of attack for constant Re and Ma numbers specifically for NACA 0012 airfoil using the bisection method.

Thanks.

Best,
Tushar

**Comment by Tushargh29** [comment]
Here, I got the following results:

1. **alpha of 3.5 degrees = non-buffet case**
2. **alpha of 4.5 degrees = buffet case**

I will mail you the animation showing the buffet for NACA 0012-34 airfoil, please have a look at it there.

Thank you.

Best,
Tushar

![nonbuffet(3 5)_0012-34](https://user-images.githubusercontent.com/71929988/115268546-ab5b6200-a13a-11eb-8cc0-95f81285e539.png)
![buffet(4 5)_0012-34](https://user-images.githubusercontent.com/71929988/115268559-ae565280-a13a-11eb-9cb7-460e67e2232e.png)

**Comment by AndreWeiner** [comment]
I suggest placing the next point at 4.0° angle of attack.
Best, Andre

**Comment by Tushargh29** [comment]
Hello sir,

I tried 4 degrees of alpha last time, I was little skeptical about the proper and complete buffet development there so I moved to 4.5 degrees, I wanted to discuss this in the meeting.

Nevertheless, even 4 degrees of alpha is showing buffet and can be seen in Paraview visualization. I found the Cl fluctuations slightly less than 4.5 case so I thought of  4.5 degrees. But that was my observation might not be correct.

Thanks.

Best,
Tushar
![buffet(4)_0012-34](https://user-images.githubusercontent.com/71929988/115388036-b5816d00-a1db-11eb-9027-483353513df1.png)
